style.css

```diff
@@ -1,417 +1,621 @@
+  flex-direction: column;
+  width: 100%;
+  height: 100%;
+  background-color: white;
+  color: #3a4a42;
+  border: 1px lightgrey solid;
+  border-radius: 5px;
 }
 
-.disabled button {
-  border: 1px solid #b4b4b4;
+.menu-btn,
+.header-nav-mobile > button {
   background-color: white;
-  padding: 6px 6px 2px 6px;
-  border-radius: 32px;
 }
 
-.card-container {
-  padding: 0 200px;
-  gap: 10px;
-  display: grid;
-  justify-content: center;
-  grid-template-columns: auto auto auto;
+.header-nav-mobile > button {
+  margin: 0.5em 0.5em;
+  align-self: flex-end;
+  cursor: pointer;
 }
 
-.project-card {
-  display: flex;
-  width: 327px;
-  border-radius: 8px;
-  background: #ebf0ee;
-  border: 1px solid #d0d9d4;
+.header-nav-mobile > img {
+  width: 2em;
 }
 
-.descriptions {
-  position: relative;
-  margin-top: 252px;
-  border-radius: 8px;
-  padding: 16px;
-  height: 222px;
-  background: #fff;
+.header-nav-mobile > ul > li {
+  margin-left: 28px;
+  margin-top: 40px;
+  font-size: 24px;
+  font-weight: 600;
+  cursor: pointer;
 }
 
-.card-title {
-  width: 295px;
-  height: 57px;
-  font-family: "Inter", sans-serif;
-  font-style: normal;
-  font-weight: 600;
+.header-nav-mobile > ul > li > a {
   font-size: 24px;
-  line-height: 32px;
-  display: flex;
-  align-items: center;
-  text-align: center;
+  font-weight: 600;
   color: #3a4a42;
+  cursor: pointer;
 }
 
-.grid-container {
-  display: grid;
-  gap: 8px;
-  margin-top: 16px;
-  grid-template-columns: auto auto auto auto;
-  list-style-type: none;
-  padding-left: 1%;
-  width: 100%;
+.link-portfolio:hover,
+.link-about:hover,
+.link-contact:hover {
+  color: var(--color-primary);
 }
 
-.itemBox {
-  display: flex;
+.menu-btn.hidden {
+  display: none;
 }
 
-.item {
-  border-radius: 4px;
-  padding: 12px 8px;
-  background: #ebf0ee;
+.header-nav-mobile.hidden {
+  display: none;
+}
 
-  /* font-family: "Inter"; */
+.header-logo {
+  text-decoration: none;
+  font-family: 'Inter', sans-serif;
   font-style: normal;
   font-weight: 600;
-  font-size: 12px;
-  text-align: center;
-  line-height: 16px;
-  letter-spacing: 0.03em;
+  font-size: 20px;
+  line-height: 28px;
+  color: #28352f;
 }
 
-.see-project-btn {
-  padding: 12px;
-  background: #36b37f;
-  border-radius: 4px;
-  position: absolute;
-  width: 122px;
-  left: calc(50% - 122px / 2 + 0.5px);
-  bottom: 24px;
+.headline {
+  background: #fff url(./img/header-illsutration-mobile.svg) no-repeat;
+  background-size: 100%;
+  overflow: hidden;
+}
 
-  /* font-family: "Inter"; */
-  font-style: normal;
-  font-weight: 500;
-  font-size: 17px;
-  line-height: 24px;
-  border: transparent;
-  color: #fff;
+.headline h1:first-child {
+  margin-top: 1.5em;
 }
 
-.divider {
-  background: #36b37f;
-  height: 4px;
-  border-radius: 5px;
-  width: 50px;
-  margin: 0 auto;
+.headline p {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 20px;
+  line-height: 28px;
+  margin: 2em 24px;
+  text-align: center;
+  color: #172b4d;
```

Comment:
- [ ] Please complete all the changes required by the previous code reviewer to approve the project. Kindly make sure that the mobile version looks similar to the Figma version for a better user interface. 👇🏽 

| Figma design | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/201496260-083001be-c93d-4918-9958-7be4217acfb4.png) | ![image](https://user-images.githubusercontent.com/58642949/201496276-35b24e2b-a3b0-40ab-a1c1-92fdfeded427.png) |